packages/network-controller/src/NetworkController.ts

```diff
@@ -311,25 +303,17 @@ export class NetworkController extends BaseControllerV2<
   }
 
   /**
-   * Sets a new configuration for web3-provider-engine.
+   * Method to inilialize the provider.
```

Comment:
[nit] I think we should try to provide information that can't already be gleaned from the name of the method itself. 

What do we think about this?

``` suggestion
   * Creates the provider and block tracker for the configured network,
   * using the provider to gather details about the network.
```